src/js/redux/actions/bulkDownload/bulkDownloadActions.js

```diff
@@ -63,3 +63,25 @@ export const handleDownloadRequest = (res) => (dispatch) => {
     dispatch(setDownloadExpectedFile(res.file_name));
     dispatch(setDownloadPending(true));
 };
+
+export const bulkPrimeAwardTypeChange = (state) => ({
```

Comment:
FYSA another pattern we moving away from is using the parameter name `state` for all our actions; instead, we are trying to use something that describes the type of state we're dealing with.

IE in this case I would expect `awardType` or something?